3 - PB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -343,920 +321,2256 @@ Questo documento è redatto in modo incrementale, così da risultare sempre conf
 
 = Use Case
 
-#set heading(numbering: (..nums) => {
-  let values = nums.pos();
-  if (values.len() > 0){
-      values.at(values.len() - 1) = values.at(values.len() - 1);
+== Introduzione
+
+In questa sezione sono presentati i casi d'uso e i relativi diagrammi relativi a WMS3. I casi d'uso sono stati definiti a seguito dell'analisi del dominio di applicazione e delle funzionalità richieste dal Proponente, durante riunioni interne ed esterne. Le modalità di definizione sono descritte in #ndp_v, alle sezioni _Processo di analisi della missione_ e _Processo di definizione di bisogni e requisiti degli stakeholder_.
+
+I casi d'uso sono organizzati secondo il dominio di appartenenza:
+- *Ambiente 3D* (@uc-ambiente): configurazione e modifica dell'ambiente 3D;
+- *Zone e Bin* (@uc-zone): operazioni CRUD sulle zone e gestione dei bin;
```

Comment:
In questo punto e in quello dopo della lista, il tag alle sezioni viene visualizzato come "Caso d'uso x.y" e non "Sezione x.y"